apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -606,6 +870,45 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     }
   end
 
+  @doc """
+  If IFE's spend is in blocks, find its txpos and update the IFE.
+  """
+  def find_ifes_in_blocks(
+        %ExitProcessor.Request{piggybacked_blocks_result: blocks} = request,
+        %__MODULE__{in_flight_exits: ifes} = state
+      ) do
+    updated_ifes =
+      ifes
+      |> Enum.filter(fn {_, ife} -> ife.tx_included_at == nil end)
+      |> Enum.map(fn {hash, ife} -> {hash, ife, find_ife_in_blocks(ife, blocks)} end)
+      |> Enum.map(fn {hash, ife, position} -> {hash, %InFlightExitInfo{ife | tx_included_at: position}} end)
+      |> Map.new()
+
+    state = %{state | in_flight_exits: Map.merge(ifes, updated_ifes)}
+    {request, state}
```

Comment:
I see we're not persisting this particular change. It may be OK, but are we conscious about this being so?

If yes, let's leave a comment somewhere (?), that this "cache-like" thing is kept in memory only.

Also, let's make a comment near where `tx_included_at` is defined, that if `nil`, it doesn't mean that it has not been included, is that (possibly) we just haven't yet put it's position to memory

Reply:
Done. Also renamed, as requested in other commit.